### PR TITLE
feat(herosection): add herosection to app

### PR DESCRIPTION
**Pull request by DorMosheZarka**
Implemented the HeroSection component within the application, providing a visually engaging introductory area.

### CODE REVIEW

File: src/components/HeroSection/HeroSection.module.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```
Comment (noammu) [inline src/components/HeroSection/HeroSection.module.css]: If you decided in the previous PRs to put these imports in the ```index.css``` file (which is the better way imo), then lets keep on doing that and add this import there.

File: src/components/HeroSection/HeroSection.module.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+.heroSection {
+   margin: 2%;
+}
```
Comment (noammu) [inline src/components/HeroSection/HeroSection.module.css]: Why percentages and not em?

File: src/components/HeroSection/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <div className={styles.heroSection}>
+         <h1>Transform Lives with a Single Gesture</h1>
+         <h2>
+            Every donation lights up a path to a brighter future. <br />
+            Join us in making a real difference where it matters most. <br />
+            Together, we can turn hope into action.
+         </h2>
+      </div>
+   );
+}
```
Comment (noammu) [inline src/components/HeroSection/HeroSection.tsx]: Why ```div``` and not ```<section>``` if it is a hero ....section? 😉

**Review by noammu — CHANGES_REQUESTED** [review]

File: src/components/HeroSection/HeroSection.module.css (hunk 2 above)
Comment (DorMosheZarka) [reply src/components/HeroSection/HeroSection.module.css]: It is a different behavior, % is more responsive a bit.

**Review by noammu — APPROVED** [review]
Excellent PR!
Do you see how fast it becomes when the PR is so focused? Imagine this code was also with the other 2-3 PRs that you had...

**Pull request merged**